build.sh

```diff
@@ -43,7 +43,7 @@ echo "Downloading installer components..."
 
 cd "$DL"
 
-wget -Nc "$PYTHON_URI"
```

Comment:
`wget -Nc $f` != `curl $f`. I would add `-C -` for `-c` (continue). For `-N` (timestamping), it seems like there's a way to wrap it with `-z` and `-r`, not sure if it's pretty. https://stackoverflow.com/a/57173943